engine/src/main/java/org/terasology/world/chunks/localChunkProvider/LocalChunkProvider.java

```diff
@@ -178,6 +191,22 @@ public ChunkViewCore getSubviewAroundChunk(Vector3i chunkPos) {
         return null;
     }
 
+    private ChunkViewCore createWorldView(BlockRegion region, Vector3i offset) {
```

Comment:
There's the same AABBi/BlockRegion issue here (and in the ChunkViewCoreImpl constructor it uses).